test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -27,3 +17,13 @@ export async function dynamicInstantiate (url) {
     }
   };
 }
+
+export function resolve(specifier, base, defaultResolver) {
+  if (builtins.has(specifier)) {
```

Comment:
Is this relying on the global `Set.prototype.has`?

Reply:
its a test, it should be safe right?

Reply:
oh lol i didn't notice that :-p yes, this is fine